samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial: Setup Redis Triggers for Azure Function in Visual Studio
+Article 08/08/2023 Nadia Bugarin, Riley MacDonald, Phoebe Owusu
+
+In this tutorial, you learn how to:
+1. Install packages on an existing Azure function
+2. Use a Redis trigger with your Azure Function
+3. Deploy an Azure Function
```

Comment:
This is a good story but these should have matching headings in the document and link to those sections.  Right now, you document isn't telling that story.